service-front/web/main.scss

```diff
@@ -61,3 +61,8 @@ $govuk-assets-path: "../assets/";
   color:  govuk-colour("brown") ;
   font-weight: bold;
 }
+.app-table__cell--no-border {
```

Comment:
This class needs a more descriptive name, since it's not doing anything with the border